hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/hive/HoodieCombineHiveInputFormat.java

```diff
@@ -879,12 +883,14 @@ public void createPool(JobConf conf, PathFilter... filters) {
       return (CombineFileSplit[]) inputSplitShims.toArray(new HadoopShimsSecure.InputSplitShim[inputSplitShims.size()]);
     }
 
+    @Override
     public HadoopShimsSecure.InputSplitShim getInputSplitShim() throws IOException {
       return new HadoopShimsSecure.InputSplitShim();
     }
 
+    @Override
     public RecordReader getRecordReader(JobConf job, CombineFileSplit split, Reporter reporter,
-        Class<RecordReader<K, V>> rrClass) throws IOException {
+                                        Class<RecordReader<K, V>> rrClass) throws IOException {
```

Comment:
Can you check the indentation is correct? Based on the "one PR one thing" convention, it would be better to revert this change. WDYT?

Reply:
Good catch, may IDE formats it automatically.